src/Components/CreateImageWizard/formComponents/Oscap.tsx

```diff
@@ -198,45 +234,46 @@ const OScapNoneOption = ({ setProfileName }) => {
   );
 };
 
-OScapNoneOption.propTypes = {
-  setProfileName: PropTypes.any,
-};
-
-const OScapSelectOption = ({ profile_id, setProfileName, input }) => {
+const OScapSelectOption = ({
+  profile_id,
+  setProfileName,
+  input,
+}: {
+  profile_id: DistributionProfileItem;
+  setProfileName: (name: string) => void;
+  input?: string;
+}) => {
   const { getState } = useFormApi();
   const { data } = useGetOscapCustomizationsQuery({
     distribution: getState()?.values?.['release'],
     profile: profile_id,
   });
-  if (
-    input &&
-    !data?.openscap?.profile_name.toLowerCase().includes(input.toLowerCase())
-  ) {
-    return null;
+
+  if (input) {
+    const profileName = data?.openscap?.profile_name;
+    if (
+      profileName &&
+      !profileName.toLowerCase().includes(input.toLowerCase())
+    ) {
+      return null;
+    }
   }
 
   return (
     <SelectOption
       key={profile_id}
       value={profile_id}
       onClick={() => {
-        setProfileName(data?.openscap?.profile_name);
+        setProfileName(data?.openscap?.profile_name || 'Default Value');
```

Comment:
I don't think adding a `Default value` literal string is the correct approach to solve this issue. As for the `None` literal string you were adding up there, it changes the logic of the component. Which is not what you want in a conversion from `js` to `tsx`. The goal being to make sure that behavior is identical.

You need to find another way to solve this one.